### ISSUE TITLE
Stylus buttons not activating

### ISSUE THREAD

**Issue by traemand** [xournalpp/xournalpp#4388]
### Discussed in https://github.com/xournalpp/xournalpp/discussions/4387

<div type='discussions-op-text'>

<sup>Originally posted by **traemand** November  6, 2022</sup>
Hi,

I've recently switched back to windows 10 from Ubuntu on my surface pro 5th gen. I've had no issues using Xournal++ while on Ubuntu, but on Windows i'm having issues. I set my button 1 to Hand tool and button 2 to Eraser, but none of them are being activated when i push them. The same setting was working on Ubuntu. I'm using an older Intuous Art Medium.

Any help is appreciated!</div>

**Comment by rolandlo** [comment]
Do you have the `Merge button events with stylus tip events` checkbox enabled under the input system tab of menu `Edit > Preferences`?

**Comment by traemand** [comment]
> Do you have the `Merge button events with stylus tip events` checkbox enabled under the input system tab of menu `Edit > Preferences`?

No, it was not. I tried it on, but it does not seem to make a difference. 
I could note that holding down the buttons actually makes the pen NOT write, although the "dot" from the tip is still visible.

**Comment by hypersad** [comment]
very very very very slow response to stylus pen

**Comment by sugampratap** [comment]
@hypersad i wasa also facing the same issue, i just followed rolandlo steps (`edit>Preferences>Merge button events with stylus tip events`) & specified the stylus buttons with desired tools in `edit>Preferences>stylus` and this worked for me.

**Comment by Xelbayria** [comment]
I've tried everything I can think of with Wacom Intous (M)'s Pen. I couldn't get it working... 

I even tried the "stylus tip events" enabled as well. It's not working. Any advice?

**Comment by rolandlo** [comment]
> 
> Devs, if you are reading this, The "Application Defined" doesn't work with Xournal++. I've tried everything I can think of. it doesn't work. You may want to see if you can make Xournal++ work with the Wacom Software.

Is there some reference on what the "Application Defined" setting changes? We probably would need to have input logs to figure it out ourselves. See [wiki](https://github.com/xournalpp/xournalpp/wiki/How-to-provide-debug-info-for-input-related-bugs)

**Comment by Xelbayria** [comment]
> Is there some reference on what the "Application Defined" setting changes? We probably would need to have input logs to figure it out ourselves. See [wiki](https://github.com/xournalpp/xournalpp/wiki/How-to-provide-debug-info-for-input-related-bugs)

"Application Defined" in Wacom Software is to use how the software define the pen. in this case, It made Wacom Software handle the setting of Pen to Xournalpp and define the buttons of the pen. 

I'll see if I can get the log using the instructions via the wiki you provided.
